src/pages/mainpage/DateBar.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import styled from "styled-components";
+
+
+const DateBar = () => {
+  const today = new Date();
+  const year = today.getFullYear(); 
+  const month = today.getMonth() + 1;
+  const date = today.getDate();
+
+  // 0(일요일)부터 6(토요일)까지 숫자로 반환
+  const week = ["일", "월", "화", "수", "목", "금", "토"];
+  const dayOfWeek = week[today.getDay()]; // 현재 요일
+
+  // 한글로 변환!!
+  const formattedDate = `${year}년 ${month}월 ${date}일 ${dayOfWeek}요일`;
+
+  return <DateContainer>{formattedDate}</DateContainer>;
+};
```

Comment:
이건 제가 사용했던 방법인데 ES6의 Intl.DateTimeFormat을 사용해 날짜를 포매팅하면 코드의 가독성을 높이고, 다국어 처리에도 용이합니다 :)

```suggestion
const DateBar = () => {
  const today = new Date();
  const formatter = new Intl.DateTimeFormat('ko-KR', {
    year: 'numeric',
    month: 'long',
    day: 'numeric',
    weekday: 'long',
  });
  const formattedDate = formatter.format(today);

  return <DateContainer>{formattedDate}</DateContainer>;
};
```